addons/stock/views/stock_picking_views.xml

```diff
@@ -194,6 +194,16 @@
                                 <span class="o_stat_text">Detailed Operations</span>
                             </div>
                         </button>
+                        <button name="action_next_transfer"
+                            class="oe_stat_button"
+                            icon="fa-exchange"
+                            type="object"
+                            help="List view of next transfers"
+                            invisible="state != 'done' or not show_next_transfers">
```

Comment:
Why not when the state isn't `done`? Might be nice to have the button when processing a chain of pull rules created pickings.